cmd/flightctl-periodic/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/flightctl/flightctl/internal/config"
+	"github.com/flightctl/flightctl/internal/periodic_checker"
+	"github.com/flightctl/flightctl/internal/store"
+	"github.com/flightctl/flightctl/pkg/log"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	log := log.InitLogs()
+	log.Println("Starting periodic")
+
+	cfg, err := config.LoadOrGenerate(config.ConfigFile())
+	if err != nil {
+		log.Fatalf("reading configuration: %v", err)
+	}
+	log.Printf("Using config: %s", cfg)
+
+	logLvl, err := logrus.ParseLevel(cfg.Service.LogLevel)
+	if err != nil {
+		logLvl = logrus.InfoLevel
+	}
+	log.SetLevel(logLvl)
+
+	log.Println("Initializing data store")
+	db, err := store.InitDB(cfg, log)
+	if err != nil {
+		log.Fatalf("initializing data store: %v", err)
+	}
```

Comment:
```suggestion
	log.Println("Initializing data store")
	db, err := store.InitDB(cfg, log)
	if err != nil {
		log.Fatalf("initializing data store: %v", err)
	}
```

Should we call to Init db from multiple places? I am worried about migrations etc, at least no two processes should attempt DB migration/creation. Is there a way to configure it so it will only wait for DB to be ready?

Reply:
Init yes, migration no